documentation/modules/oauth/proc-oauth-authorization-keycloak-example.adoc

```diff
@@ -256,14 +277,34 @@ keytool -keystore /tmp/truststore.p12 -storetype pkcs12 -alias sso -storepass $S
 KAFKA_HOST_PORT=my-cluster-kafka-bootstrap:9093
 STOREPASS=storepass
 
-echo "Q" | openssl s_client -showcerts -connect $KAFKA_HOST_PORT 2>/dev/null | awk ' /BEGIN CERTIFICATE/,/END CERTIFICATE/ { print $0 } ' > /tmp/my-cluster-kafka.crt
+echo "Q" | openssl s_client -showcerts -connect $KAFKA_HOST_PORT 2>/dev/null | awk ' /BEGIN CERTIFICATE/,/END CERTIFICATE/ { print $0 } ' > /tmp/my-cluster-kafka.pem
 ----
 
+If the client accesses the Kafka cluster over the Ingress or the OpenShift Route the Kafka bootstrap hostname will be different, and you may need to specify it as an extra argument to the `openssl` command in order to get the certificate containing the proper SNA records:
```

Comment:
In such case, the port will be different as well ... and so it will be with node ports, but then you do not need the TLS-SNI of course.

Reply:
How should this be fixed to be correct?

Reply:
You need to explain how to find the proper bootstrap address.

Reply:
I can add a link to: https://strimzi.io/docs/operators/latest/configuring.html#proc-accessing-kafka-using-nodeports-str